<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="statusTip">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Gaze speed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="gaze_counter_spinbox">
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="left_gaze_label">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>110</height>
         </size>
        </property>
        <property name="text">
         <string>Eye Gaze Left</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="b1_1">
          <property name="text">
           <string>Wheel Chair</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="b1_3">
          <property name="text">
           <string>Message</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="b2_2">
          <property name="text">
           <string>Music</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="b3_1">
          <property name="text">
           <string>Light</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="b4_1">
          <property name="text">
           <string>Browser</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QComboBox" name="selectMethodComboBox">
        <property name="currentText">
         <string notr="true" extracomment="None"/>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="maxVisibleItems">
         <number>5</number>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::NoInsert</enum>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="main_image_label">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>400</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>WheelChair Speed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="chairSpeedSlider">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="value">
           <number>50</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="invertedControls">
           <bool>false</bool>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBothSides</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Blink Speed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="blink_counter_spinbox">
          <property name="value">
           <number>3</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="right_gaze_label">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>110</height>
         </size>
        </property>
        <property name="text">
         <string>Eye Gaze Right</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="b1_2">
          <property name="text">
           <string>SMS</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="b2_1">
          <property name="text">
           <string>Video</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="b2_3">
          <property name="text">
           <string>Documents</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="b3_2">
          <property name="text">
           <string>Fan</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="b4_2">
          <property name="text">
           <string>Emergency</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
